functions/github/github.js

```diff
@@ -156,25 +156,29 @@ async function getLibraryMetadata(
     libraryChanges,
 ) {
   const libraryMetadata = {};
+  const allLibraryNames = [...libraryNames, ...Object.keys(libraryChanges)]
+      .filter((value, index, self) => self.indexOf(value) === index);
   const libraryVersions = await getLibraryVersions(
       octokit,
       releaseBranchName,
-      libraryNames,
+      allLibraryNames,
   );
-  log("Fetched library versions", {libraryVersions: libraryVersions});
 
   for (const library in libraryVersions) {
     if (Object.prototype.hasOwnProperty.call(libraryVersions, library)) {
+      const libraryIsReleasing = libraryNames.includes(library);
+      const libraryHasChanges = libraryChanges[library] &&
+        libraryChanges[library].length > 0;
       libraryMetadata[library] = {
         "updatedVersion": libraryVersions[library],
-        "optedIn": !libraryChanges[library],
+        "optedIn": libraryIsReleasing && !libraryHasChanges,
```

Comment:
Why the `!libraryHasChanges`?

Reply:
We infer that a library was opted in to release if it's releasing but does not have any changes (i.e. we found no changes in the release report, since the release report did not include this library initially).

Reply:
I noticed after looking at this again that my logic here isn't exactly right. I fixed here https://github.com/firebase/firebase-release-dashboard/pull/30/commits/c8b8ecb64c5c9ab5731d75b072f14aef9d51f846